.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: [
+    'plugin:react/recommended',
+    'standard'
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 11,
+    sourceType: 'module'
+  },
+  plugins: [
+    'react'
```

Comment:
Is this for the JS SDK api docs?

Reply:
Yep, the whole thing is rendered in React and without this plugin, it complains about imported but not used variables.

Reply:
You're correct, it affects only docs.